langchain4j/src/main/java/dev/langchain4j/classification/TextClassifier.java

```diff
@@ -12,13 +13,45 @@
  */
 public interface TextClassifier<E extends Enum<E>> {
```

Comment:
```suggestion
public interface TextClassifier<L> {
```

Shouldn't it support `String` as requested in the issue?

Reply:
Done.